### PR TITLE
[Merged by Bors] - chore: remove unnecessary cdots

### PR DESCRIPTION
**Pull request by adomani**
These `·` are scoping when there is a single active goal.

These were found using a modification of the linter at #12339.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Nat/Pairing.lean [hunk 1]
```diff
@@ -132,8 +132,8 @@ theorem pair_lt_pair_left {a₁ a₂} (b) (h : a₁ < a₂) : pair a₁ b < pair
   · simp at h₁
     simp only [not_lt_of_gt (lt_of_le_of_lt h₁ h), ite_false]
     apply add_lt_add
-    exact Nat.mul_self_lt_mul_self h
-    apply Nat.add_lt_add_right; assumption
+    · exact Nat.mul_self_lt_mul_self h
```
Comment (grunweg) [inline Mathlib/Data/Nat/Pairing.lean:135]: This is adding cdots (but looks like a good change regardless).

File: Mathlib/Data/Fintype/Pi.lean [hunk 2]
```diff
@@ -125,7 +125,7 @@ theorem Fintype.piFinset_univ {α : Type*} {β : α → Type*} [DecidableEq α]
 -- but those don't work with subsingletons in lean4 as-is so we cannot do this here.
 noncomputable instance _root_.Function.Embedding.fintype {α β} [Fintype α] [Fintype β] :
   Fintype (α ↪ β) :=
-  by classical. exact Fintype.ofEquiv _ (Equiv.subtypeInjectiveEquivEmbedding α β)
+  by classical exact Fintype.ofEquiv _ (Equiv.subtypeInjectiveEquivEmbedding α β)
```
Comment (grunweg) [inline Mathlib/Data/Fintype/Pi.lean:128]: That's a nice "c"dot :-)

**Review by grunweg — APPROVED** [review]
Thank you for doing this! The code changes look good to me; I trust you/the linter that all these instances actually have only a single goal active.

File: Mathlib/Data/Nat/Pairing.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/Data/Nat/Pairing.lean:135]: You are right: I messed up with git and pulled in a change from the other PR that was *adding* the cdots.

File: Mathlib/Data/Fintype/Pi.lean (hunk 2 above)
Comment (adomani) [reply Mathlib/Data/Fintype/Pi.lean:128]: That's a plus of looking at the `SyntaxNodeKind`, rather than what the user actually typed...

**Comment by fpvandoorn** [comment]
Thanks!

bors merge
bors d+

**Pull request closed without merging**